std_msgs/Header header
paul_vision/BBox3d[] boxes
#original pos
# float64 x
# float64 y
# float64 z